src/validation/version.ts

```diff
@@ -0,0 +1,11 @@
+export function isNotValidWeaviateVersion(version: string) {
+  if (typeof version === 'string') {
+    const versionNumbers = version.split('.');
+    if (versionNumbers.length >= 2) {
+      const major = parseInt(versionNumbers[0], 10);
+      const minor = parseInt(versionNumbers[1], 10);
+      return major <= 1 && minor < 16;
+    }
+  }
+  return false;
+}
```

Comment:
Sorry for not being clearer in the original review but I still think this is confusing since a double negative is involved in the logic & naming convention here. Here we are saying "Is the Weaviate version invalid" and return "No" on a success, which is easy to get confused about due to the double negative

I think naming it `isValidWeaviateVersion`, returning `true` on line 10, and refactoring line 7 to return `false` is preferable. Perhaps the following on line 7:
```typescript
return major <= 1 && minor < 16 ? false : true;
```

Then the conditional statements in `path.ts` and `beaconPath.ts` would be:
```typescript
if(!isValidWeaviateVersion(support.version)) { ... }
```

In this way double negatives are avoided and the code is easier to read 😁 The logic of saying "Is not valid" in the conditional statements is still clear from the `!` operator also